### PR TITLE
feat(docs): update tweet links to embed posts

### PR DESCRIPTION
**Pull request by realdreamer**
closes 
https://app.asana.com/0/1205288273408141/1205310320676409/f
https://app.asana.com/0/1205288273408141/1205310320676405/f

### CODE REVIEW

File: packages/apps/docs/src/pages/docs/blogchain/2018/consensus2018-5-lessons-from-your-first-crypto-conference-day-1-2018-04-15.md [hunk 1]
```diff
@@ -12,42 +12,36 @@ layout: blog
...
-[line to registration](https://twitter.com/mdudas/status/996070779434528769)
-that represents the fundamental problems of throughput and congestion for
-blockchain as a whole.
+Lesson #1: Prepare for a line to registration that represents the fundamental
+problems of throughput and congestion for blockchain as a whole.
 
-TODO: embed
-https://twitter.com/dickerson_des/status/996041484007886848?ref_src=twsrc%5Etfw%7Ctwcamp%5Etweetembed%7Ctwterm%5E996041484007886848%7Ctwgr%5Ef54a655ebe4bf6e4f65fb28adac49d5e1911205a%7Ctwcon%5Es1_&ref_url=https%3A%2F%2Fcdn.embedly.com%2Fwidgets%2Fmedia.html%3Ftype%3Dtext2Fhtmlkey%3Da19fcc184b9711e1b4764040d3dc5c07schema%3Dtwitterurl%3Dhttps3A%2F%2Ftwitter.com%2Fdickerson_des%2Fstatus%2F996041484007886848image%3Dhttps3A%2F%2Fi.embed.ly%2F1%2Fimage3Furl3Dhttps253A252F252Fpbs.twimg.com252Fprofile_images252F873291993627193344252FpruiILEs_400x400.jpg26key3Da19fcc184b9711e1b4764040d3dc5c07
+TODO: not working
+[line to registration](https://x.com/mdudas/status/996070779434528769)
```
Comment (realdreamer) [inline packages/apps/docs/src/pages/docs/blogchain/2018/consensus2018-5-lessons-from-your-first-crypto-conference-day-1-2018-04-15.md:19]: looks like this tweet deleted/moved. 

shall I remove it cc @sstraatemans

File: packages/apps/docs/src/pages/docs/blogchain/2023/chainweaver-2-2-3-improving-user-experience-with-quicksign-support-2023-02-10.md [hunk 2]
```diff
@@ -29,15 +29,16 @@ smart contracts and push their applications into production by deploying their
...
+[demo](https://twitter.com/BlockchainDoug/status/1539733933801316361)
 
 ![Quicksign API works by sending multiple transactions to Kadena’s blockchain simultaneously](/assets/blog/0_AiU-isvNBfs5RpkP.png)
 
 We’ve already begun seeing builders and projects capitalizing on this update and
 improving their project’s UX. [**X-Wallet**](https://xwallet.kaddex.com/), a
 robust and user-friendly web-extension wallet that is native to Kadena, has
-implemented
+implemented **TODO: this tweet not working**
 [quicksign](https://twitter.com/XWalletOfficial/status/1622624077683871746)
```
Comment (realdreamer) [inline packages/apps/docs/src/pages/docs/blogchain/2023/chainweaver-2-2-3-improving-user-experience-with-quicksign-support-2023-02-10.md:42]: this tweet also seems moved/deleted

File: packages/apps/docs/src/pages/docs/blogchain/2018/consensus2018-5-lessons-from-your-first-crypto-conference-day-1-2018-04-15.md (hunk 1 above)
Comment (sstraatemans) [reply packages/apps/docs/src/pages/docs/blogchain/2018/consensus2018-5-lessons-from-your-first-crypto-conference-day-1-2018-04-15.md:19]: yeah if deleted, lets remove it

File: packages/apps/docs/src/pages/docs/blogchain/2023/chainweaver-2-2-3-improving-user-experience-with-quicksign-support-2023-02-10.md (hunk 2 above)
Comment (sstraatemans) [reply packages/apps/docs/src/pages/docs/blogchain/2023/chainweaver-2-2-3-improving-user-experience-with-quicksign-support-2023-02-10.md:42]: yup..remove it

**Review by sstraatemans — APPROVED** [review]

**Pull request merged**